config/config.example.yml

```diff
@@ -115,45 +71,6 @@ sync_permissions:
           - name: "{synched_file}"
             type: workspace
     # [Required] This section defines an array of permissions mapping between services
-    # Each item found in the `permissions_mapping` uses the following format :
-    #     resource_key1 : <permissions1> <mapping> resource_key2 : <permissions2>
-    # <permissionsX> is defined as a single permission or a list of permissions :
-    #     `permission | [permission1, permission2, ...]`
-    # <mapping> is defined as a unidirectional or bidirectional arrow : `-> | <- | <->`
-    #
-    # The permissions can either be a single permission or a list of permissions. Each of those permissions can either
-    # use an implicit format (`<name>` or `<name>-match`) or an explicit format (`<name>-<access>-<scope>`).
-    # When using an implicit format, if the access and/or scope are not specified, it will use the default
-    # access `allow` and/or the default scope `recursive`.
-    # For more info on Magpie permissions :
-    #     https://pavics-magpie.readthedocs.io/en/latest/permissions.html#permission-definition-and-modifiers
-    #     https://pavics-magpie.readthedocs.io/en/latest/permissions.html#permissions-representation
```

Comment:
Leave minimally the "format / schema" portion + references. Then add the extra reference to cowbird readthedocs with everything below with further details / examples.